src/data/set/lattice.lean

```diff
@@ -39,6 +39,11 @@ instance : distrib_lattice (set α) :=
 lemma monotone_image {f : α → β} : monotone (image f) :=
 assume s t, assume h : s ⊆ t, image_subset _ h
 
+lemma image_inter_subset (f : α → β) (s t : set α) :
+  f '' (s ∩ t) ⊆ f '' s ∩ f '' t :=
+show f '' (s ⊓ t) ≤ f '' s ⊓ f '' t,
+from monotone.map_inf_le monotone_image _ _
```

Comment:
There is a lemma `image_inter` in `data/set/basic`, and last line of its proof actually proves exactly `image_inter_subset`. Do you mind moving it there?